app/assets/javascripts/app/components/comment_form/templates/comment_form.html.slim

```diff
@@ -2,12 +2,16 @@
   .TopBarLayout-header
     md-toolbar on-scroll="" on-scroll-anchor="#js-comment-content" scrolling-class="md-whiteframe-z1"
       md-content.md-padding layout="row" layout-align="space-between center"
-        md-button.md-primary.md-warn md-no-ink="" ng-click="toggle()" layout="row" layout-align="center center"
+        md-button.md-primary.md-warn md-no-ink="" ng-click="ctrl.toggle()" layout="row" layout-align="center center"
           .IconClose
           | hide
 
-        council-raised-button.Button.Button--secondary ng-click="create(comment)"
+        council-raised-button.Button.Button--secondary ng-click="ctrl.createComment()" ng-hide="ctrl.comment.id"
           | Add Comment
+
+        council-raised-button.Button.Button--secondary ng-click="ctrl.updateComment()" ng-show="ctrl.comment.id"
+          | Update Comment
+
   .TopBarLayout-content
     .Page.Page--white.u-fullSize
-      textarea.Editor placeholder="State your arguments" ng-model="comment.body" id="js-comment-content"
+      textarea.Editor placeholder="State your arguments" ng-model="ctrl.comment.body" id="js-comment-content"
```

Comment:
```textarea#js-comment-content.Editor placeholder="State your arguments" ng-model="ctrl.comment.body"```

in case you miss what i did, you can move write the id like that in *slim*